doc/developers-guide/Testing-MongooseIM.md

```diff
@@ -402,6 +402,4 @@ Finally, set `GDRIVE_PARENT_DIR` environment variable of your build to the direc
 ### Load testing
 
 Alongside CI, we do also CLT (Continuous Load Testing).
-We have our own load testing infrastructure, called Tide, which is triggered after every successful test run, and gives us a feedback on changes to MongooseIM performance.
-
-Test results are publicly available on the [Hello Tide!](http://tide.erlang-solutions.com/public) page.
+We use [amoc](https://github.com/esl/amoc) and [amoc-arsenal-xmpp](https://github.com/esl/amoc-arsenal-xmpp/) to perform tests that give us a feedback on changes to MongooseIM performance.
```

Comment:
Maybe need highlight MongooseIM maybe `MongooseIM` or **MongooseIM**

Reply:
I don't think that's a convention used anywhere in the docs regarding the name MongooseIM - is it really needed?

Reply:
I suppose it's make sense to highlight `MongooseIM`, but up to you, this is not a critical issue, this is just proposing.

Reply:
I must admit I'm not a fan of it - "MongooseIM" is already capitalised, and it is the name of the program the documentation is about, no real need to highlight it there any more. If we had some different font for the name that we used everywhere, or something like a different way of writing the name then I would be more for it.
Maybe we can think about this change in the context of some other PR, as it would probably entail changing "MongooseIM" throughout the docs.